poly-commitment/src/tests/batch_15_wires.rs

```diff
@@ -83,10 +81,19 @@ where
             let mut start = Instant::now();
             let comm = (0..a.len())
                 .map(|i| {
+                    let n = a[i].len();
+                    let num_chunks = if n % srs.g.len() == 0 {
```

Comment:
@dannywillems could you please give a bit of context on what's happening with the tests? Primarily, were they committing to less chunks than necessary, and if yes, does it mean that the tests were not sound -- they were passing but not doing what was originally expected? 

In this particular case, it seems that `a[i].len()` is always `1`, isn't it?..

Reply:
[This](https://github.com/o1-labs/proof-systems/pull/2010#discussion_r1529949745) should answer your question

Reply:
That seems to be very orthogonal, my question here was about the test:

The length of the vector in `a` is random up to `s = length_i`. Elements in `length_i` are up to `500`. So unless the SRS is less than `2^8` we will need to create one chunk, and so this code would behave exactly as before. But the SRS is `2^7` above, so was the test actually /failing/ if the SRS was smaller? We created some commitments, they are not correct, but does the test check it after construction-time later at all?